### ISSUE TITLE
nosetests falls into infinite try catch loop

### ISSUE THREAD

**Issue by arthurmensch** [nistats/nistats#61]
RUnning nosetests nistats yields infinite:
```
Data absent, downloading...
Error uncompressing file: [Uncompress] unknown archive file format: /home/arthur/nilearn_data/spm_auditory/sub001/MoAEpilot.zip
Archive corrupted, trying to download it again.
fM00223/fM00223_004.img missing from filelist!
```

I reckon there is an ionvalid url there, and I guess retrying infinitly is not a desired behaviour.

**Comment by bthirion** [comment]
I guess that we should rely on the Nilearn machinery to improve this behavior.

**Comment by arthurmensch** [comment]
with ```_fetch_files``` ?

**Comment by bthirion** [comment]
On 25/02/2016 11:10, Arthur Mensch wrote:
>
> with |_fetch_files| ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/nistats/nistats/issues/61#issuecomment-188704069>.
>
Yes.

**Comment by bthirion** [comment]
Fixed in #74.
